cmd/kubeadm/app/cmd/init.go

```diff
@@ -197,6 +197,8 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
 	flagSet.Var(utilflag.NewMapStringString(&cfg.APIServerExtraArgs), "apiserver-extra-args", "A set of extra flags to pass to the API Server or override default ones in form of <flagname>=<value>")
 	flagSet.Var(utilflag.NewMapStringString(&cfg.ControllerManagerExtraArgs), "controller-manager-extra-args", "A set of extra flags to pass to the Controller Manager or override default ones in form of <flagname>=<value>")
 	flagSet.Var(utilflag.NewMapStringString(&cfg.SchedulerExtraArgs), "scheduler-extra-args", "A set of extra flags to pass to the Scheduler or override default ones in form of <flagname>=<value>")
+
+	flagSet.BoolVar(&cfg.NoTaintMaster, "no-taint-master", cfg.NoTaintMaster, "Do not taint the master node.")
```

Comment:
@errordeveloper - ping here, b/c you were making the case for it, but there is a push to try and not add any new flags.  /cc @luxas

Reply:
I added the flags after it was requested in https://github.com/kubernetes/kubernetes/pull/55479#discussion_r161392303, but I deliberately kept it as a separate commit which could easily be dropped.

Reply:
I agree with the push back on new flags, so perhaps we could add just the config file filed for now.